tests/python_package_test/test_dask.py

```diff
@@ -97,6 +98,57 @@ def test_classifier(output, centers, client, listen_port):
     assert_eq(p1_proba, p2_proba, atol=0.3)
 
 
+@pytest.mark.parametrize('output', data_output)
+@pytest.mark.parametrize('centers', data_centers)
+def test_classifier_pred_contrib(output, centers, client, listen_port):
+    X, y, w, dX, dy, dw = _create_data('classification', output=output, centers=centers)
+
+    dask_classifier = dlgbm.DaskLGBMClassifier(
+        time_out=5,
+        local_listen_port=listen_port,
+        tree_learner='data'
+    )
```

Comment:
Add `n_estimators=10` and `num_leaves=10`?
#3786.

Reply:
oh good idea

Reply:
added in https://github.com/microsoft/LightGBM/pull/3774/commits/64285895ec11c4a23af838c7cdfe62889a78b1a5

Reply:
ok I just added this again (was lost because of a bad merge conflict resolution, sorry)